web/packages/teleterm/src/ui/ClusterConnect/ClusterLogin/FormLogin/FormPasswordless/FormPasswordless.tsx

```diff
@@ -61,7 +61,7 @@ const StyledPaswordlessBtn = styled(ButtonText)`
   &:hover,
   &:active,
   &:focus {
-    border-color: ${({ theme }) => theme.colors.action.active};
+    border-color: red;
```

Comment:
It feels like we should use a value from the theme here, no?

Reply:
This was a test I forgot to revert, good catch